pyproject.toml

```diff
@@ -212,6 +212,7 @@ addopts = [
 ]
 filterwarnings = [
     "error::ResourceWarning",
+    "error::DeprecationWarning",
```

Comment:
This will catch *future* DeprecationWarning cases and turn them into exception in your test suite. Do you want this?

Reply:
Discussed at stand-up, but recording it here: I think it's worthwhile to keep this to ensure that any DeprecationWarnings are immediately brought to our attention, so we can file PRs to stay current. If an occasion arises where fixing the deprecated code is too large for a simple PR, we can either file a PR to xfail some tests or remove this line.